parm/params_grib2_tbl_new

```diff
@@ -294,7 +294,6 @@
    0   7   204   1 EFHL
    0   3   222   1 EFSH
    0   7     9   0 EHLX
-   2   0   237   1 EIWATER
```

Comment:
Does this intended to be removed?

The same are LAKEFRC and LANDFRAC.

There are new added code for GOES 18 ABI. But why to remove those definition that are not related to GOES ABI. A results from sync with RRFS fix?

Reply:
Yes, those removals are from syncing with RRFS fix.  As we discussed yesterday, it seems the UPP repository params_grib2_tbl_new file is updated to include more variables than in our realtime file, but we can deal with the differences when we sync with EMC.